.vscode/settings.json

```diff
@@ -5,8 +5,8 @@
   "files.autoSave": "onFocusChange",
   "files.insertFinalNewline": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll": true,
-    "source.sortImports": true
+    "source.fixAll": "explicit",
```

Comment:
Why?

Reply:
Sorry, I didn't even touch that file. Whenever, I work on the project this file changes automatically, I forgot to revert back to its default.
I will fix it and push it again.